src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -47,27 +55,40 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
-	public String changeTheFirstLetterToUppercase(String word) {
-
-		String[] regexResults = word.split("_");
+	public void getDataFromFile() {
 
+		File file = new File("D:/java/europe_capitals.txt");
 		StringBuilder stringBuilder = new StringBuilder();
+		String dataFromFile = null;
 
-		if (regexResults.length == 1) {
+		try (BufferedReader buffer = new BufferedReader(new FileReader(file))) {
 
-			String wordAfterChange = (stringBuilder.append(word.substring(0, 1).toUpperCase())
-					.append(word.substring(1))).toString();
-			return wordAfterChange;
-		} else {
+			while (buffer.ready()) {
 
-			String wordAfterChange = (stringBuilder.append(regexResults[0].substring(0, 1).toUpperCase())
-					.append(regexResults[0].substring(1))).append("_")
-							.append(regexResults[1].substring(0, 1).toUpperCase()).append(regexResults[1].substring(1))
-							.toString();
+				stringBuilder.append(buffer.readLine()).append(" ");
+			}
 
-			return wordAfterChange;
+		} catch (IOException e1) {
+
+			e1.printStackTrace();
 		}
 
+		dataFromFile = stringBuilder.toString();
+		regexResults = dataFromFile.split(" ");
+
+	}
+
+	public boolean checkKey(String key) {
```

Comment:
`checkKey` to mało precyzyjna nazwa metody. Proponuję `existsByKey`

Reply:
OK, poprawię. Ja zaproponowałbym checkExistenceByKey. Czy taka nazwa też może być?

Reply:
Nie, `existsBy...` to nazwa zgodna z konwencją JPA.

Reply:
OK, poprawione.